src/Libraries/CoreNodes/DSCoreNodesImages.resx

```diff
@@ -5119,44 +5119,6 @@
         gOTfKGA1R7m+SSE8APXA8bsfLqezAAAAAElFTkSuQmCC
 </value>
   </data>
-  <data name="DSCore.IO.File.ReadText.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
But only the obsolete nodes have been moved to a different namespace, and obsolete nodes are not supposed to be shown in the library view. So why modifying obsolete name should require changing image used by library view?